app/renderer/src/components/Inspector/Source.jsx

```diff
@@ -20,27 +21,98 @@ const Source = (props) => {
     methodCallInProgress,
     mjpegScreenshotUrl,
     isSourceRefreshOn,
+    pageSourceSearchText,
     t,
   } = props;
 
+  /**
+   * Highlights the part of the node text in source tree that matches the search term.
+   */
+  const highlightNodeMatchingSearchTerm = (nodeText, searchText) => {
+    if (!searchText) {
+      return nodeText;
```

Comment:
It is not very clear to me why we return the nodeText if no matches have been found. I would expect nothing to be returned/highlighted in such case